client/src/main/scala/grasshopper/client/parser/model/ParsedAddress.scala

```diff
@@ -1,14 +1,8 @@
 package grasshopper.client.parser.model
 
-case class AddressPart(
-  addressNumber: String,
-  city: String,
-  state: String,
-  streetName: String,
-  zip: String
-)
-case class ParsedAddress(input: String, parts: AddressPart)
-
 object ParsedAddress {
-  def empty: ParsedAddress = ParsedAddress("", AddressPart("", "", "", "", ""))
+  def empty: ParsedAddress = ParsedAddress("", List.empty)
 }
+
+case class AddressPart(`type`: String, value: String)
```

Comment:
Agreed.  By the time I was at this point, I just wanted to get through making it work in the geocoder, and not do a bunch of refactoring in the parser.  I will change this attribute name, probably to "code".